3_visualize.R

```diff
@@ -16,6 +19,44 @@ p3_targets_list <- list(
              format = "file",
              pattern = map(p2_inst_data_hourly)),
   
+  # Create and save log file containing data availability summary
+  tar_target(
+    p3_sitelist_summary_csv,
+    summarize_site_list(p2_site_list_csv,p1_daily_data,p1_inst_data,fileout = "3_visualize/log/sitelist_summary.csv"),
+    format = "file"),
+  
+  # Create and save indicator file for NWIS daily data
+  tar_target(
+    p3_daily_timeseries_ind_csv,
+    command = save_target_ind_files("3_visualize/log/daily_timeseries_ind.csv",names(p3_daily_timeseries_png)),
+    format = "file"),
+  
+  # Create and save indicator file for NWIS instantaneous data
+  tar_target(
+    p3_inst_timeseries_ind_csv,
+    command = save_target_ind_files("3_visualize/log/inst_timeseries_ind.csv",names(p3_hourly_timeseries_png)),
+    format = "file"),
+  
+  # Create and save indicator file for WQP data
+  tar_target(
+    p3_wqp_ind_csv,
+    command = save_target_ind_files("3_visualize/log/wqp_data_ind.csv","p2_wqp_data_subset_csv"),
+    format = "file"),
```

Comment:
similar to above, I think it would be more intuitive to have this log file in `2_process/log`